.laminas-ci.json

```diff
@@ -1,4 +1,7 @@
 {
+    "extensions": [
+        "mcrypt"
```

Comment:
Still needed?

Reply:
yes, `mcrypt` is **not** one of the default extensions in the `continuous-integration-action` dependencies list.

[7.4 dependencies](https://github.com/laminas/laminas-continuous-integration-action/blob/1.14.x/setup/php/7.4/dependencies)
[8.0 dependencies](https://github.com/laminas/laminas-continuous-integration-action/blob/1.14.x/setup/php/8.0/dependencies)
[8.1 dependencies](https://github.com/laminas/laminas-continuous-integration-action/blob/1.14.x/setup/php/8.1/dependencies)

Reply:
should we add `"ext-mcrypt": "*"` in `composer.json`?

Reply:
Probably best to phase out mcrypt, although I must say that this patch changes scope then 😁